fe/fe-core/src/main/java/org/apache/doris/catalog/DistributionInfo.java

```diff
@@ -40,12 +40,28 @@ public enum DistributionInfoType {
     @SerializedName(value = "type")
     protected DistributionInfoType type;
 
+    @SerializedName(value = "bucketNum")
+    protected int bucketNum;
```

Comment:
Same suggestions as `DistributionDesc`

Reply:
hmm, I only move it from HashDistributionInfo && RandomDistributionInfo to base class DistributionInfo. read/write methos are compatible. I test with old fe and this one